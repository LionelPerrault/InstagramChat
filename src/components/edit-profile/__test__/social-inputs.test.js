import React from 'react'
import { create } from 'react-test-renderer'
import SocialInputs from '../social-inputs'

describe('Social-Inputs Component', () => {
  const props = {
    inputs: {
      instagram: 'https://www.instagram.com/_faiyaz_shaikh',
      github: 'https://github.com/yTakkar/',
      twitter: 'https://twitter.com/shtakkar',
      facebook: 'https://www.facebook.com/profile.php?id=100009110960262',
      website: '',
      phone: '',
    },
    change: jest.fn(),
  }

  it('should match snapshot', () => {
    const tree = create(<SocialInputs {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
