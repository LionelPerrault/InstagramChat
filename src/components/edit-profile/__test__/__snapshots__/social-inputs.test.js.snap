// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social-Inputs Component should match snapshot 1`] = `
<div
  className="edit_sm_div"
>
  <span
    className="edit_span"
  >
    Connections
  </span>
  <input
    autoComplete="false"
    disabled={false}
    maxLength="255"
    onChange={[Function]}
    placeholder="Instagram"
    spellCheck="false"
    type="text"
    value="https://www.instagram.com/_faiyaz_shaikh"
  />
  <input
    autoComplete="false"
    disabled={false}
    maxLength="255"
    onChange={[Function]}
    placeholder="Github"
    spellCheck="false"
    type="text"
    value="https://github.com/yTakkar/"
  />
  <input
    autoComplete="false"
    disabled={false}
    maxLength="255"
    onChange={[Function]}
    placeholder="Twitter"
    spellCheck="false"
    type="text"
    value="https://twitter.com/shtakkar"
  />
  <input
    autoComplete="false"
    disabled={false}
    maxLength="255"
    onChange={[Function]}
    placeholder="Facebook"
    spellCheck="false"
    type="text"
    value="https://www.facebook.com/profile.php?id=100009110960262"
  />
  <input
    autoComplete="false"
    disabled={false}
    maxLength="255"
    onChange={[Function]}
    placeholder="Website"
    spellCheck="false"
    type="text"
    value=""
  />
  <input
    autoComplete="false"
    disabled={false}
    maxLength="10"
    onChange={[Function]}
    placeholder="Phone"
    spellCheck="false"
    type="text"
    value=""
  />
</div>
`;
